### ISSUE TITLE
Missing requirements: pathlib and gdal

### ISSUE THREAD

**Issue by adammcmaster** [michaelaye/planetpy#3]
On a fresh install (on Debian Jesse) I had to manually install pathlib and gdal to use this.

**Comment by michaelaye** [comment]
Thanks for the issue!
With pathlib, pls note I don't develop for Python 2 (not enough resource for that and indicated in setup.py classifiers), and Python 3 has it as a built-in.
I totally forgot about gdal, and it's quite too big a dependency. I'm wondering how you HAD to install it though? What did you use that triggered its need?
In a29260da9886f3df4ef679c9b2c48cbbc19e685b I added a try/except to just check if it's there, but don't demand it, as it's still difficult to install cleanly sometimes.

As pathlib is readily available on pypi, maybe I should just add it as a requirement though? I guess that's all I would have to do for make it easier to work on Py2? I just can't guarantee it though.

**Comment by adammcmaster** [comment]
The [Planet Four data import script](https://github.com/zooniverse/planet-four/blob/master/data-import/fetch_metadata.py) works fine in Python 2 with pathlib installed, though I haven't tried anything beyond that. It looks like `pip install pathlib` with Python 3 still installs it even though it's not needed, so I'm not certain which is the best option there.

Here's how to reproduce the gdal issue with a fresh Debian/Python 3 installation in Docker:

```
$ docker run -it --rm python:3 bash
root@b48c53a3dd00:/# pip install planetpy
Collecting planetpy
  Downloading planetpy-0.2.2-py2.py3-none-any.whl
Collecting lxml (from planetpy)
  Downloading lxml-3.7.2-cp36-cp36m-manylinux1_x86_64.whl (7.2MB)
    100% |████████████████████████████████| 7.2MB 148kB/s
Collecting pandas (from planetpy)
  Downloading pandas-0.19.2-cp36-cp36m-manylinux1_x86_64.whl (18.9MB)
    100% |████████████████████████████████| 18.9MB 53kB/s
Collecting html5lib (from planetpy)
  Downloading html5lib-0.999999999-py2.py3-none-any.whl (112kB)
    100% |████████████████████████████████| 122kB 1.0MB/s
Collecting beautifulsoup4 (from planetpy)
  Downloading beautifulsoup4-4.5.3-py3-none-any.whl (85kB)
    100% |████████████████████████████████| 92kB 1.3MB/s
Collecting pvl (from planetpy)
  Downloading pvl-0.2.0.tar.gz (1.4MB)
    100% |████████████████████████████████| 1.4MB 43kB/s
Collecting python-dateutil>=2 (from pandas->planetpy)
  Downloading python_dateutil-2.6.0-py2.py3-none-any.whl (194kB)
    100% |████████████████████████████████| 194kB 899kB/s
Collecting pytz>=2011k (from pandas->planetpy)
  Downloading pytz-2016.10-py2.py3-none-any.whl (483kB)
    100% |████████████████████████████████| 491kB 596kB/s
Collecting numpy>=1.7.0 (from pandas->planetpy)
  Downloading numpy-1.11.3-cp36-cp36m-manylinux1_x86_64.whl (15.7MB)
    100% |████████████████████████████████| 15.7MB 92kB/s
Collecting six (from html5lib->planetpy)
  Downloading six-1.10.0-py2.py3-none-any.whl
Requirement already satisfied: setuptools>=18.5 in /usr/local/lib/python3.6/site-packages (from html5lib->planetpy)
Collecting webencodings (from html5lib->planetpy)
  Downloading webencodings-0.5.tar.gz
Installing collected packages: lxml, six, python-dateutil, pytz, numpy, pandas, webencodings, html5lib, beautifulsoup4, pvl, planetpy
  Running setup.py install for webencodings ... done
  Running setup.py install for pvl ... done
Successfully installed beautifulsoup4-4.5.3 html5lib-0.999999999 lxml-3.7.2 numpy-1.11.3 pandas-0.19.2 planetpy-0.2.2 pvl-0.2.0 python-dateutil-2.6.0 pytz-2016.10 six-1.10.0 webencodings-0.5
root@b48c53a3dd00:/# python
Python 3.6.0 (default, Dec 28 2016, 23:09:04)
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from planetpy import pdstools as pds
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/planetpy/pdstools.py", line 13, in <module>
    from . import utils
  File "/usr/local/lib/python3.6/site-packages/planetpy/utils.py", line 2, in <module>
    import gdal
ModuleNotFoundError: No module named 'gdal'
```

**Comment by michaelaye** [comment]
Ok, thanks I believe I fixed that missing dependency by just allowing it to be absent. I forgot that pdstools imports the utils, sorry.

**Comment by michaelaye** [comment]
fixed by 195ce5855fcf52aea829a00cd82fc1caae84b74f
